src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -34,7 +34,7 @@ public InCanvasSearchControl()
             {
                 Application.Current.Deactivated += currentApplicationDeactivated;
             }
-            Unloaded += InCanvasSearchControl_Unloaded; ;
+            Unloaded += InCanvasSearchControl_Unloaded;
```

Comment:
👍

Reply:
Wow build will pass in this case?

Reply:
Yes, but I bet it creates a code smell :)